swagger/src/main/scala/org/http4s/rho/swagger/SwaggerModelsBuilder.scala

```diff
@@ -228,8 +228,8 @@ private[swagger] class SwaggerModelsBuilder(formats: SwaggerFormats) {
     Operation(
       tags        = pathStr.split("/").filterNot(_ == "").headOption.getOrElse("/") :: Nil,
       summary     = collectSummary(rr),
-      consumes    = rr.validMedia.toList.map(_.renderString),
-      produces    = rr.responseEncodings.toList.map(_.renderString),
+      consumes    = rr.validMedia.toList.map(_.toString),
+      produces    = rr.responseEncodings.toList.map(_.toString),
```

Comment:
I feel like the old `renderString` function was not the same as `toString` on a MediaRange. (Possibly not non an responseEncoding`)... `toString`'s results have a `MediaRange` and some parens...

Reply:
I'm looking at this diff in http4s to see the change: https://github.com/http4s/http4s/commit/f2ca17540cde57d0e5664ac5a9e9db37b92612c1